setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 [flake8]
 application-import-names = orderable
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D105,D203,D204
```

Comment:
It's mostly missing docstrings we ignore, along with whether there are blank lines before/after class docstrings.